internal/conf/validation/cody.go

```diff
@@ -30,6 +38,27 @@ func completionsConfigValidator(q conftypes.SiteConfigQuerier) conf.Problems {
 		problems = append(problems, "'completions.enabled' has been superceded by 'cody.enabled', please migrate to the new configuration.")
 	}
 
+	allModelIds := []modelId{
+		{value: completionsConf.ChatModel, field: "chatModel"},
+		{value: completionsConf.FastChatModel, field: "fastChatModel"},
+		{value: completionsConf.CompletionModel, field: "completionModel"},
+	}
+	var modelIdsToCheck []modelId
+	for _, modelId := range allModelIds {
+		if modelId.value != "" {
+			modelIdsToCheck = append(modelIdsToCheck, modelId)
+		}
+	}
+
+	//check for bedrock ARNs
+	if completionsConf.Provider == string(conftypes.CompletionsProviderNameAWSBedrock) {
+		for _, modelId := range modelIdsToCheck {
+			if strings.HasPrefix(modelId.value, "arn:aws:") {
```

Comment:
Just for clarity, if you think this is unnecessary feel free to ignore.
```suggestion
			// When using provisioned capacity we expect an admin would just put the ARN
			// here directly, but we need both the model AND the ARN. Hence the check.
			if strings.HasPrefix(modelId.value, "arn:aws:") {
```